ffi/derive/src/util.rs

```diff
@@ -231,13 +231,13 @@ fn gen_derived_method_sig(field: &syn::Field, derive: Derive) -> syn::Signature
 
     match derive {
         Derive::Setter => parse_quote! {
-            #field_vis fn #method_name(&mut self, #field)
+            fn #method_name(&mut self, #field)
```

Comment:
So I see that visibility is not present in `syn::Signature` and thus it's not needed, but I wonder how it typechecked before

Reply:
So as we discussed with @mversic in pm, before we get lucky because all fields with getters/setters was private and thus there `#field_vis` was `None`.
A accidentally noticed that when wrote test with `pub` field and getter.

Reply:
Maybe something like property testing could have prevented that from happening in the first place. As @appetrosyan mentioned, we're in high need of better testing harnesses overall and for macros especially.

Reply:
Proptest can't test macros directly. Unfortunately it would have been of little use here.